pkgs/tools/misc/rmlint/default.nix

```
@@ -36,6 +37,11 @@ stdenv.mkDerivation rec {
   patches = [
     # pass through NIX_* environment variables to scons.
     ./scons-nix-env.patch
+    # fix #664
+    (fetchpatch {
+      url = "https://github.com/sahib/rmlint/commit/f0ca57ec907f7199e3670038d60b4702d1e1d8e2.patch";
+      hash = "sha256-Z1q7MdBxMEYtfYhfdjYIBZK64zEhvvd8V2S+Jvp0naw=";
```

```suggestion
      hash = "sha256-715X+R2BcQIaUV76hoO+EXPfNheOfw4OIHsqSoruIUI=";
```

```
error: hash mismatch in fixed-output derivation '/nix/store/ajhyjg4i7c50mwg5zary9a2fr7lkxqwb-f0ca57ec907f7199e3670038d60b4702d1e1d8e2.patch.drv':
         specified: sha256-Z1q7MdBxMEYtfYhfdjYIBZK64zEhvvd8V2S+Jvp0naw=
            got:    sha256-715X+R2BcQIaUV76hoO+EXPfNheOfw4OIHsqSoruIUI=
error: 1 dependencies of derivation '/nix/store/jx6jjlcdyqmvbzrjdcxwkn7zqhgfpwzb-rmlint-2.10.2.drv' failed to build
```

The derivation outputs the build failure because the hash of this patch is wrong.

---

I'm really confused.  This is the hash I fetched using `nix store prefetch-file https://github.com/sahib/rmlint/commit/f0ca57ec907f7199e3670038d60b4702d1e1d8e2.patch` and this builds on my machine...

---

It works now for me with the updated hash